hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/hdds/scm/pipeline/TestPipelineClose.java

```diff
@@ -204,6 +205,7 @@ public void testPipelineCloseWithPipelineAction() throws Exception {
   }
 
   @Test
+  @Ignore
```

Comment:
I suggest "linking" the issue as the reason for disabling it:

```suggestion
  @Ignore("HDDS-5604")
```

Reply:
Thanks - I have made this change.